src/classBased/components/TodoContainer.js

```diff
@@ -0,0 +1,96 @@
+import React from "react"
+import TodosList from "./TodosList"
+import Header from "../../functionBased/components/Header";
+import InputTodo from "./InputTodo";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
Great job by importing the react components

Reply:
Thank you!!!